shared/prometheus/metrics/__init__.py

```
@@ -0,0 +1,40 @@
+    "get_backend_http_metrics",
+    "reset_backend_http_metrics",
+    "BACKEND_HTTP_DURATION_BUCKETS",
+    # Chat Shell HTTP metrics
+    "ChatShellHTTPMetrics",
+    "get_chat_shell_http_metrics",
+    "reset_chat_shell_http_metrics",
+    "CHAT_SHELL_HTTP_DURATION_BUCKETS",
+    # WebSocket metrics
+    "WebSocketMetrics",
+    "get_websocket_metrics",
+]
```

Ruff (0.14.14)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)